packages/legend-studio/src/stores/ApplicationConfig.ts

```diff
@@ -98,7 +98,13 @@ class ApplicationCoreOptions {
    * to figure out how we want to store this element in SDLC.
    */
   EXPERIMENTAL__enableFullGrammarImportSupport = false;
-
+  /**
+   * Allows disabling of resolving element paths inside a RawLambda
+   *
+   * NOTE: when we move to save imports as part of the user's project, this feature
+   * will no longer be needed and can be removed
+   */
```

Comment:
Maybe add one more line here saying we should use it with the section index flag, as they are in the same group. Though I agree, for unforeseeable reasons, it makes sense to split them like this.